qiskit_sphinx_theme/furo/base/page.html

```diff
@@ -32,7 +32,9 @@
     <div class="header-left">
       <label class="nav-overlay-icon" for="__navigation">
         <div class="visually-hidden">Toggle site navigation sidebar</div>
-        <i class="icon"><svg><use href="#svg-menu"></use></svg></i>
+        {#- QISKIT CHANGE: start. Replace left table of contents with text. -#}
+          <h4> Site contents </h4><i class="icon"><svg><use href="#svg-right-chevron"></use></svg></i> 
```

Comment:
Bump that this should be called something like qiskit-chevron-right-black

Reply:
that is not a chevron, is the first menu icon

Reply:
@Eric-Arellano  the chevron, I called right-chevron

Reply:
What I mean is that we want our custom icons to all have the word `qiskit` in them. It helps to make more clear what is our own custom code vs. the default Furo.

Reply:
I'm confused by that, though? It is a "chevron" icon, like that little arrow thing. What's the issue with `qiskit-chevron-right`?